deepcell_toolbox/utils.py

```diff
@@ -58,20 +59,25 @@ def erode_edges(mask, erosion_width):
         ValueError: mask.ndim is not 2 or 3
     """
     if erosion_width:
-        new_mask = np.zeros(mask.shape)
         if mask.ndim == 2:
-            strel = disk(erosion_width)
+            new_mask = np.copy(mask)
+            for i in range(erosion_width):
+                boundaries = find_boundaries(new_mask, mode='inner')
+                new_mask[boundaries > 0] = 0
+            return new_mask
         elif mask.ndim == 3:
+            new_mask = np.zeros(mask.shape)
             strel = ball(erosion_width)
+            for cell_label in np.unique(mask):
+                if cell_label != 0:
+                    temp_img = mask == cell_label
+                    temp_img = binary_erosion(temp_img, strel)
+                    new_mask = np.where(mask == cell_label, temp_img, new_mask)
```

Comment:
Does `find_boundaries` only work for 2D images? I just copied/pasted code from the ndim==2 block and it passes the tests. I did not visualize anything however.